src/curl_json.c

```diff
@@ -310,6 +304,11 @@ static int cj_cb_string (void *ctx, const unsigned char *val,
   return (cj_cb_number (ctx, (const char *) val, len));
 } /* int cj_cb_string */
 
+static int cj_cb_boolean (void * ctx, int boolVal)
+{
+  return (cj_cb_number (ctx, (boolVal)? "1":"0", 1));
```

Comment:
Nit, at your option: I'd slightly prefer to assign to a variable and call `strlen()` on that instead of hard-coding the `1`:

```c
char const *n = boolVal ? "1" : "0";
return (cj_cb_number (ctx, n, strlen (n));
```

Reply:
exactly - you're doing a function call, which does actual work, which is costly.  Thus only putting in 0 is a safe guess here. 

maybe not making it an implicit If, and writing `return cj_cb_number (ctx, "1", 1); else ...` would make it more readable...

Reply:
Yes, it would always return 1, but it would make it clear to the reader that the length of the string is passed and not, for example, a boolean "true" argument.

I'd be quite surprised if this would result in a function call. I'm fairly sure that compilers figure out that it's always invoked with a string literal.

Anyway, it's an optional comment, so there's no point in stressing this ;)

Best regards,
—octo